webextensions/api/action.json

```diff
@@ -238,7 +238,11 @@
               },
               "edge": "mirror",
               "firefox": {
-                "version_added": "101"
+                "version_added": "101",
+                "notes": [
+                  "Must be called from inside the handler for a <a href='https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/User_actions'>user action</a>. From Firefox 108, this restriction has been lifted behind a preference. See <a href='https://bugzil.la/1799344'>bug 1799344</a>.",
```

Comment:
Ideally, I'd just shorten this to "Since Firefox 108, user interaction is no longer required to call this API."

Now, since the pref is required, maybe also `... when the ... pref is set (<a ...>bug xxx</a>).`

When the pref is finally flipped, we could do something like this:
https://github.com/mdn/browser-compat-data/blob/2687c37c6450a1c4c323370cb9bb1a486416fbd4/webextensions/api/identity.json#L63-L67

... or just keep the minimal simple text from the top of this comment, without any reference to a preference, since the preference (flag) would be irrelevant: https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#removal-of-irrelevant-flag-data

Reply:
Thanks for the feedback, I pushed a change :)